app/views/curation_concerns/file_sets/_single_use_link_rows.html.erb

```diff
@@ -0,0 +1,16 @@
+<% single_use_links.each_with_index do |link, index| %>
+  <% type = link.path =~ /downloads/ ? 'Download' : 'Show' %>
```

Comment:
Good call -- yes, these would be great as presenter methods. Like I said, though, I don't feel very strongly about this point if you start going down this road and it becomes annoying. ;)